2018-06-25 18:48:03.936208: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-06-25 18:48:05.148119: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1344] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.62
pciBusID: 0000:02:00.0
totalMemory: 10.92GiB freeMemory: 228.56MiB
2018-06-25 18:48:05.148196: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1423] Adding visible gpu devices: 0
2018-06-25 18:48:05.663688: I tensorflow/core/common_runtime/gpu/gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-06-25 18:48:05.663783: I tensorflow/core/common_runtime/gpu/gpu_device.cc:917]      0 
2018-06-25 18:48:05.663801: I tensorflow/core/common_runtime/gpu/gpu_device.cc:930] 0:   N 
2018-06-25 18:48:05.664048: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1041] Created TensorFlow device (/device:GPU:0 with 174 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:02:00.0, compute capability: 6.1)
2018-06-25 18:48:05.666226: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 174.56M (183042048 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
Using TensorFlow backend.
WARNING:tensorflow:From /opt/conda/envs/python36/lib/python3.6/site-packages/tensorflow/contrib/learn/python/learn/datasets/base.py:198: retry (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
Instructions for updating:
Use the retry module or similar alternatives.
2018-06-25 18:48:12.368827: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1423] Adding visible gpu devices: 0
2018-06-25 18:48:12.368933: I tensorflow/core/common_runtime/gpu/gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-06-25 18:48:12.368950: I tensorflow/core/common_runtime/gpu/gpu_device.cc:917]      0 
2018-06-25 18:48:12.368961: I tensorflow/core/common_runtime/gpu/gpu_device.cc:930] 0:   N 
2018-06-25 18:48:12.369145: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1041] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 174 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:02:00.0, compute capability: 6.1)
2018-06-25 18:48:12.874363: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:12.879671: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:12.883571: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:12.885711: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:12.887730: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:12.889472: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:12.895547: E tensorflow/stream_executor/cuda/cuda_dnn.cc:403] could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
2018-06-25 18:48:12.895583: F tensorflow/core/kernels/conv_ops.cc:712] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms) 
sh: Zeile 1: 29884 Abgebrochen             python MNIST\ Novelty\ Detection.py $i >> log_mnist_$i.txt
2018-06-25 18:48:14.819185: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-06-25 18:48:16.189810: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1344] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.62
pciBusID: 0000:02:00.0
totalMemory: 10.92GiB freeMemory: 228.56MiB
2018-06-25 18:48:16.189913: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1423] Adding visible gpu devices: 0
2018-06-25 18:48:16.654259: I tensorflow/core/common_runtime/gpu/gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-06-25 18:48:16.654407: I tensorflow/core/common_runtime/gpu/gpu_device.cc:917]      0 
2018-06-25 18:48:16.654430: I tensorflow/core/common_runtime/gpu/gpu_device.cc:930] 0:   N 
2018-06-25 18:48:16.654759: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1041] Created TensorFlow device (/device:GPU:0 with 174 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:02:00.0, compute capability: 6.1)
2018-06-25 18:48:16.656718: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 174.56M (183042048 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
Using TensorFlow backend.
WARNING:tensorflow:From /opt/conda/envs/python36/lib/python3.6/site-packages/tensorflow/contrib/learn/python/learn/datasets/base.py:198: retry (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
Instructions for updating:
Use the retry module or similar alternatives.
2018-06-25 18:48:23.241674: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1423] Adding visible gpu devices: 0
2018-06-25 18:48:23.241801: I tensorflow/core/common_runtime/gpu/gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-06-25 18:48:23.241817: I tensorflow/core/common_runtime/gpu/gpu_device.cc:917]      0 
2018-06-25 18:48:23.241827: I tensorflow/core/common_runtime/gpu/gpu_device.cc:930] 0:   N 
2018-06-25 18:48:23.242023: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1041] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 174 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:02:00.0, compute capability: 6.1)
2018-06-25 18:48:23.764186: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:23.770454: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:23.773921: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:23.775514: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:23.777456: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:23.779048: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:23.790244: E tensorflow/stream_executor/cuda/cuda_dnn.cc:403] could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
2018-06-25 18:48:23.790325: F tensorflow/core/kernels/conv_ops.cc:712] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms) 
sh: Zeile 1: 30253 Abgebrochen             python MNIST\ Novelty\ Detection.py $i >> log_mnist_$i.txt
2018-06-25 18:48:26.429881: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-06-25 18:48:27.859209: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1344] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.62
pciBusID: 0000:02:00.0
totalMemory: 10.92GiB freeMemory: 228.56MiB
2018-06-25 18:48:27.859535: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1423] Adding visible gpu devices: 0
2018-06-25 18:48:28.326590: I tensorflow/core/common_runtime/gpu/gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-06-25 18:48:28.326756: I tensorflow/core/common_runtime/gpu/gpu_device.cc:917]      0 
2018-06-25 18:48:28.326808: I tensorflow/core/common_runtime/gpu/gpu_device.cc:930] 0:   N 
2018-06-25 18:48:28.327104: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1041] Created TensorFlow device (/device:GPU:0 with 174 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:02:00.0, compute capability: 6.1)
2018-06-25 18:48:28.329248: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 174.56M (183042048 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
Using TensorFlow backend.
WARNING:tensorflow:From /opt/conda/envs/python36/lib/python3.6/site-packages/tensorflow/contrib/learn/python/learn/datasets/base.py:198: retry (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
Instructions for updating:
Use the retry module or similar alternatives.
2018-06-25 18:48:37.550949: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1423] Adding visible gpu devices: 0
2018-06-25 18:48:37.551092: I tensorflow/core/common_runtime/gpu/gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-06-25 18:48:37.551115: I tensorflow/core/common_runtime/gpu/gpu_device.cc:917]      0 
2018-06-25 18:48:37.551131: I tensorflow/core/common_runtime/gpu/gpu_device.cc:930] 0:   N 
2018-06-25 18:48:37.551400: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1041] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 174 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:02:00.0, compute capability: 6.1)
2018-06-25 18:48:38.373135: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:38.377519: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:38.382866: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:38.385568: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:38.388993: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:38.391954: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:38.400580: E tensorflow/stream_executor/cuda/cuda_dnn.cc:403] could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
2018-06-25 18:48:38.400665: F tensorflow/core/kernels/conv_ops.cc:712] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms) 
sh: Zeile 1: 30411 Abgebrochen             python MNIST\ Novelty\ Detection.py $i >> log_mnist_$i.txt
2018-06-25 18:48:41.521314: I tensorflow/core/platform/cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: SSE4.1 SSE4.2 AVX AVX2 FMA
2018-06-25 18:48:42.929164: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1344] Found device 0 with properties: 
name: GeForce GTX 1080 Ti major: 6 minor: 1 memoryClockRate(GHz): 1.62
pciBusID: 0000:02:00.0
totalMemory: 10.92GiB freeMemory: 228.56MiB
2018-06-25 18:48:42.929268: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1423] Adding visible gpu devices: 0
2018-06-25 18:48:43.372739: I tensorflow/core/common_runtime/gpu/gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-06-25 18:48:43.372827: I tensorflow/core/common_runtime/gpu/gpu_device.cc:917]      0 
2018-06-25 18:48:43.372845: I tensorflow/core/common_runtime/gpu/gpu_device.cc:930] 0:   N 
2018-06-25 18:48:43.373066: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1041] Created TensorFlow device (/device:GPU:0 with 174 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:02:00.0, compute capability: 6.1)
2018-06-25 18:48:43.374612: E tensorflow/stream_executor/cuda/cuda_driver.cc:936] failed to allocate 174.56M (183042048 bytes) from device: CUDA_ERROR_OUT_OF_MEMORY
Using TensorFlow backend.
WARNING:tensorflow:From /opt/conda/envs/python36/lib/python3.6/site-packages/tensorflow/contrib/learn/python/learn/datasets/base.py:198: retry (from tensorflow.contrib.learn.python.learn.datasets.base) is deprecated and will be removed in a future version.
Instructions for updating:
Use the retry module or similar alternatives.
2018-06-25 18:48:52.008596: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1423] Adding visible gpu devices: 0
2018-06-25 18:48:52.008823: I tensorflow/core/common_runtime/gpu/gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-06-25 18:48:52.008846: I tensorflow/core/common_runtime/gpu/gpu_device.cc:917]      0 
2018-06-25 18:48:52.008864: I tensorflow/core/common_runtime/gpu/gpu_device.cc:930] 0:   N 
2018-06-25 18:48:52.090014: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1041] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 174 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:02:00.0, compute capability: 6.1)
2018-06-25 18:48:52.930171: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:52.933998: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:52.939063: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:52.941856: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:52.945250: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:52.948172: E tensorflow/stream_executor/cuda/cuda_blas.cc:462] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-06-25 18:48:52.958849: E tensorflow/stream_executor/cuda/cuda_dnn.cc:403] could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
2018-06-25 18:48:52.958902: F tensorflow/core/kernels/conv_ops.cc:712] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms) 
sh: Zeile 1: 30770 Abgebrochen             python MNIST\ Novelty\ Detection.py $i >> log_mnist_$i.txt
